tests/admin/test_system_queries.py

```diff
@@ -98,8 +125,6 @@ def test_sudo_queries(sudo_query: str, expected: bool) -> None:
         or is_query_alter(sudo_query)
         or is_query_optimize(sudo_query)
     ) == expected
```

Comment:
should you be passing this into `is_valid_query` as well to ensure it gives the correct output?